app/main.py

```diff
@@ -1,13 +1,20 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
```

Comment:
your code looks good but the curly braces must be located in one of two options: open and start with the text or have a line break between the text.
You can see examples here:
https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md

Reply:
Ok! Solved! )